index.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
     target.options = target.options || {};
 
     // Build all paths
-    var bulmaPath = path.join(target.bowerDirectory, 'bulma');
```

Comment:
@akagomez If we're going to allow this to used in nested addon structure we should prob just say

```js
var bulmaPath;
if (!bowerDirectory) {
  bulmaPath = path.join('bower_components', 'bulma');
} else {
  bulmaPath = path.join(target.bowerDirectory, 'bulma');
}
```